src/db/migrations/2019-001-create-users.js

```diff
@@ -22,12 +22,12 @@ export default {
     },
     email: {
       type: Sequelize.STRING,
-      allowNull: false,
+      allowNull: true,
```

Comment:
why this change?

Reply:
Some facebook users do not have an email attached. For the direct login, validations would prevent signup when email is missing.

Reply:
make sense!